RecoTauTag/RecoTau/plugins/PFTauMiniAODPrimaryVertexProducer.cc

```diff
@@ -72,6 +72,9 @@ void PFTauMiniAODPrimaryVertexProducer::nonTauTracksInPVFromPackedCands(
     const reco::Track *track = cand.bestTrack();
     if (track == nullptr)
       continue;
+    //MB: Skip tracks with ill-defined momentum (why it happens?)
+    if (!std::isfinite(track->pt()) || !std::isfinite(track->eta()) || !std::isfinite(track->phi()))
```

Comment:
is a check on cand.pt() going to be enough?
This looks like something that was fixed in 106X/master in #28043 / #27988 

it would be computationally cheaper to check the candidate (and just pt) rather than to make a track out of it first and then check the bad value.

Reply:
I see an inf with simple TTree::Scanning of `patPackedCandidates_lostTracks__PAT.obj.pt()` in the provided file for `EventAuxiliary.event()==1556024`. So, it seems like a simple check of the candidate pt should be enough.

Reply:
I have checked how tracks are built from `pat::PackedCandaites` and indeed it is fine to check momentum of a candidate before retrieving its track. I also agree that check of one coordinate (pt) should be enough. Therefore, I'll simplify the code.